pom.xml

```diff
@@ -211,6 +212,8 @@
                   <exec executable="git" dir="${project.basedir}" failonerror="true">
                     <arg value="add" />
                     <arg value="." />
+                    <!-- Don't add (unrelated) not yet tracked files -->
+                    <arg value="--update" />
```

Comment:
Uses the [`--update` option](https://git-scm.com/docs/git-add#Documentation/git-add.txt---update) to not add untracked files. The Maven release plugin does not seem to detect those for its clean worktree check and this could otherwise lead to unrelated files being added accidentally.

Using this option should be fine because these steps for replacing version references are only expected to affect already tracked files.

Reply:
OK